### PR TITLE
Changed sidebar color on hover

### PR DESCRIPTION
**Pull request by SebastianGode**

### CODE REVIEW

File: otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css [hunk 1]
```diff
@@ -313,10 +313,10 @@ body {
 }
 .docs-sidebar-toc a {
   text-decoration: none;
-  color: var(--main-body-color);
+  color: var(--dt-color-black);
 }
 .docs-sidebar-toc a:hover {
-  color: var(--dt-color-blue-50);
+  color: var(--dt-color-primary-hover);
```
Comment (gtema) [inline otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:319]: please not this color - it is far away from being attractive

Comment (gtema) [reply otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:319]: blue should be fine, but not magenta. I do not really like it at all

Comment (gtema) [reply otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css:323]: that is not an enforcement to do it this way. I.e. Yam is not using it this way (on sidebars). It doesn't look nice. Especially on the right sidebar this is really ugly.

**Review by tischrei — APPROVED** [review]

**Comment by SebastianGode** [comment]
Let's keep that PR open till we have voted on this.

**Review by gtema — APPROVED** [review]
Don't like it, but ok

**Pull request merged**